app/views/demographics/new.html.erb

```diff
@@ -0,0 +1,56 @@
+<h2 class="text-center">Demographics</h2>
+<div class="row">
+  <%= form_tag demographics_path do %>
+    <div class="well">
+      <p class="text-center">To help us make sure we provide all students with equal opportunity, you have the option of disclosing demographic information about yourself.</p>
+      <p class="text-center">Filling this out is voluntary and, if you choose not to, you will not be negatively affected in any way.</p>
+      <hr>
+      <div class="form-group">
+        <%= label_tag :genders, "Gender (check all that apply):" %>
+        <ul class="list-unstyled">
+          <% ["Female", "Male", "Non-binary", "Transgender"].each do |gender| %>
```

Comment:
all of these choices aren't DRY with the model. i think there's a nice way to get them in using `form_for` and `include ActiveModel::Model` on the model, or something like that. check out `https://github.com/epicodus/epicenter/blob/master/app/models/attendance_record_amendment.rb` for an example of active model (although this doesn't show how to use it with `form_for`). also check out https://robots.thoughtbot.com/activemodel-form-objects.